src/brouwers/kitreviews/models.py

```diff
@@ -21,28 +24,28 @@ class KitReview(models.Model):
     raw_text = models.TextField(
         _(u'review'),
         help_text=_('This is your review. You can use BBCode here.')
-        )
+    )
     html = models.TextField(blank=True, help_text=u'raw_text with BBCode rendered as html')
-    positive_points = models.TextField(_('positive points'), blank=True)
-    negative_points = models.TextField(_('negative points'), blank=True)
-    rating = models.PositiveSmallIntegerField(_(u'rating'), default=DEFAULT_RATING)
+    prop = models.ManyToManyField('KitReviewProperty', blank=True, related_name='reviews')
+    rating = models.PositiveSmallIntegerField(_(u'rating'), default=DEFAULT_RATING,
+                                              validators=[MinValueValidator(MIN_RATING), MaxValueValidator(MAX_RATING)])
 
     # linking to extra information
     album = models.ForeignKey(Album, verbose_name=_('album'), blank=True, null=True)
     topic_id = models.PositiveIntegerField(
```

Comment:
Have a look at https://github.com/modelbrouwers/modelbrouwers/blob/develop/src/brouwers/groupbuilds/models.py#L90 to properly store topic ID's. That field comes with a widget that accepts an integer or a url to a topic.